stream-chat-android-offline/src/test/java/io/getstream/chat/android/livedata/controller/ChannelUnreadCountDomainTest.kt

```diff
@@ -18,20 +16,23 @@ internal class ChannelUnreadCountDomainTest : BaseDisconnectedIntegrationTest()
     fun testUnreadCount() {
         val messages = listOf(data.message1, data.message2Older)
```

Comment:
Should I delete this file?

Reply:
Isn't those tests useful?

Reply:
I reworked `computeUnread` and wrote tests for it in `ComputeUnreadCountTest`